docker.sh

```diff
@@ -2,80 +2,12 @@
 
 TASK_NAME=$1;
 
-export PROJECT_NAME=$(cat ./.project_name)
-export PROJECT_XDEBUG_ENABLED=false
-export PROJECT_WEB_DIR=${PROJECT_WEB_DIR:="web"}
-export PROJECT_INDEX_FILE=${PROJECT_INDEX_FILE:="index.php"}
-export PROJECT_DEV_INDEX_FILE=${PROJECT_DEV_INDEX_FILE:="index_dev.php"}
-export APP_NAME=$(echo $(cat composer.json | grep name | head -1 | awk -F: '{ print $2 }' | sed 's/[",\r]//g' | tr -dc '[:alnum:]\n\r' | tr '[:upper:]' '[:lower:]'))
-export APP_VERSION=$(echo $(cat composer.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[",\r]//g'))
-export USERID=$(id -u);
-echo "User ID: $USERID";
-echo -e "\nIMAGE VERSION: $APP_NAME:$APP_VERSION\n";
-declare -i BUILD_STATUS=0;
+__DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-function imageExists {
-    IMAGE_NAME=$1
-    if docker history -q "$IMAGE_NAME" > /dev/null 2>&1; then
-        echo - "$IMAGE_NAME already exist"
-        return 1;
-    fi
 
-    return 0;
-}
-
-function buildImages {
-    NAME=$1
-    VERSION=$2
-    USERID=${3:-1000}
-    PHP=${4-:"all"}
-
-    imageExists "${NAME}:${VERSION}-php56xdebug"
-    if [[ $? == 0 ]] && [[ "$PHP" == "php56xdebug" || "$PHP" == "all" ]]; then
-        docker build -t "${NAME}:${VERSION}-php56xdebug" --build-arg "USERID=$USERID" docker/php56xdebug
-    fi
-
-    imageExists "${NAME}:${VERSION}-php7xdebug"
-    if [[ $? == 0 ]] && [[ "$PHP" == "php7xdebug" || "$PHP" == "all" ]]; then
-        docker build -t "${NAME}:${VERSION}-php7xdebug" --build-arg "USERID=$USERID" docker/php7xdebug
-    fi
-
-    imageExists "${NAME}:${VERSION}-php71xdebug"
-    if [[ $? == 0 ]] && [[ "$PHP" == "php71xdebug" || "$PHP" == "all" ]]; then
-        docker build -t "${NAME}:${VERSION}-php71xdebug" --build-arg "USERID=$USERID" docker/php71xdebug
-    fi
-
-    imageExists "${NAME}:${VERSION}-nginx"
-    if [[ $? == 0 ]]; then
-        docker build -t "${NAME}:${VERSION}-nginx" docker/nginx
-    fi
-}
-
-function runBuild {
-    export IMAGE_VERSION=$1
-    BUILD_OUTPUT=$(docker-compose up php)
-    docker-compose kill > /dev/null 2>&1
-    docker-compose rm -f -v > /dev/null 2>&1
-
-    echo -e "$BUILD_OUTPUT";
-    if echo "$BUILD_OUTPUT" | grep -q "exited with code 0"; then
-        BUILD_STATUS=0;
-    else
-        BUILD_STATUS=1;
-    fi
-
-}
-
-function runInBackground {
-    export IMAGE_VERSION=$1
-
-    docker-compose -f docker-compose.yml -f docker-compose.local.yml kill > /dev/null 2>&1
-    docker-compose -f docker-compose.yml -f docker-compose.local.yml rm -f -v > /dev/null 2>&1
-    docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d nginx php mysql
-    docker-compose -f docker-compose.yml -f docker-compose.local.yml exec php bash
-    docker-compose -f docker-compose.yml -f docker-compose.local.yml kill > /dev/null 2>&1
-    docker-compose -f docker-compose.yml -f docker-compose.local.yml rm -f -v > /dev/null 2>&1
-}
+for FILE in $(ls "${__DIR}/bash") ; do
```

Comment:
Order is not important because those scripts contain only functions.
Calling those functions is below in this file, when everything is included.